cscs-checks/libraries/gridtools/gridtools_base.py

```diff
@@ -0,0 +1,194 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.parameterized_test(['vertical_advection_dycore_naive'],
+                        ['vertical_advection_dycore_mc'],
+                        ['simple_hori_diff_naive'], ['simple_hori_diff_mc'],
+                        ['vertical_advection_dycore_cuda'],
+                        ['simple_hori_diff_cuda'])
+class GridToolsCheck(rfm.RegressionTest):
+    def __init__(self, variant):
+        super().__init__()
+
+        # Check if this is a device check
+        is_cuda_test = "cuda" in variant
+
+        self.descr = 'GridTools test base'
+
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.modules = ['CMake/3.12.4', 'Boost', 'gcc/5.3.0']
+        if is_cuda_test:
+            self.modules.append(
+                'cudatoolkit/9.2.148_3.19-6.0.7.1_2.1__g3d9acc8')
+        self.sourcesdir = 'https://github.com/GridTools/gridtools.git'
+        self.build_system = 'CMake'
+
+        self.build_system.config_opts = [
+            '-DBoost_NO_BOOST_CMAKE="true"',
+            '-DCMAKE_BUILD_TYPE:STRING=Release',
+            '-DBUILD_SHARED_LIBS:BOOL=ON',
+            '-DGT_GCL_ONLY:BOOL=OFF',
+            '-DCMAKE_CXX_COMPILER=CC',
+            '-DGT_USE_MPI:BOOL=OFF',
+            '-DGT_SINGLE_PRECISION:BOOL=OFF',
+            '-DGT_ENABLE_PERFORMANCE_METERS:BOOL=ON',
+            '-DGT_TESTS_ICOSAHEDRAL_GRID:BOOL=OFF',
+            '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
+            '-DBOOST_ROOT=$BOOST_ROOT',
+            '-DGT_ENABLE_PYUTILS=OFF',
+            '-DGT_TESTS_REQUIRE_FORTRAN_COMPILER=ON',
+            '-DGT_TESTS_REQUIRE_C_COMPILER=ON',
+            '-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON']
+
+        if is_cuda_test:
+            self.build_system.config_opts.extend((
+                '-DGT_ENABLE_BACKEND_X86:BOOL=OFF',
+                '-DGT_ENABLE_BACKEND_NAIVE:BOOL=OFF',
+                '-DGT_ENABLE_BACKEND_MC=OFF',
+                '-DGT_ENABLE_BACKEND_CUDA:BOOL=ON',
+                '-DCUDA_ARCH:STRING=sm_60',
+                '-DCMAKE_CUDA_HOST_COMPILER:STRING=CC'))
+        else:
+            self.build_system.config_opts.extend((
+                '-DGT_ENABLE_BACKEND_X86:BOOL=ON',
+                '-DGT_ENABLE_BACKEND_NAIVE:BOOL=ON',
+                '-DGT_ENABLE_BACKEND_MC=ON',
+                '-DGT_ENABLE_BACKEND_CUDA:BOOL=OFF'))
+
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        if is_cuda_test:
+            self.num_gpus_per_node = 1
+            self.num_tasks = 1
+        else:
+            self.valid_systems.append('daint:mc', 'dom:mc')
```

Comment:
change to `self.valid_systems.extend(['daint:mc', 'dom:mc'])`

Reply:
`+=` is preferred, cos it's more readable.